opa-manager-operator/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+name: gatekeeper-manager
```

Comment:
This charm should be named gatekeeper-controller-manager to match upstream terminology.

Reply:
changed